metaserver/src/test/scala/tests/compiler/CompilerSuite.scala

```diff
@@ -14,6 +16,10 @@ class CompilerSuite extends MegaSuite {
       Nil
   )
 
+  def toDocument(name: String, code: String): Document = {
+    InteractiveSemanticdb.toDocument(compiler, code, name, 10000)
```

Comment:
What is this number?

Reply:
```scala
/**
@param timeout max number of milliseconds to allow the presentation compiler
*/
```

from: https://github.com/scalameta/scalameta/blob/00e7bb0683ad646b20c72482976e2b1f651da2b5/scalameta/semanticdb-scalac-core/src/main/scala/scala/meta/interactive/InteractiveSemanticdb.scala#L47

Reply:
I have some WIP experiments to use monix Task to avoid ad-hoc timeouts and support proper cancellation for presentation compiler requests